core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2821,4 +2821,14 @@ package object config {
       .version("4.0.0")
       .timeConf(TimeUnit.MILLISECONDS)
       .createOptional
+
+  private[spark] val SPARK_API_MODE =
+    ConfigBuilder("spark.api.mode")
+      .doc("For Spark Classic applications, specify whether to automatically use Spark Connect by" +
+        "running a local Spark Connect server. The value can be `classic` or `connect`.")
```

Comment:
shall we mention that is local spark connect server is dedicated to this application and will be shut down after the application completes?